views/index.html

```diff
@@ -15,36 +15,26 @@
     <div class="hero hero--full text-center">
       <div class="container">
         <img class="hero__logo" src="/img/fflogo.png" alt="Firefox Logo"  />
-        <h1 class="hero__title">Extension Compatibility Test for Firefox</h1>
+        <h1 class="hero__title">Extension Compatibility Test for Firefox is closing!</h1>
```

Comment:
Should we say "closing" or "moving" here?

Reply:
Not really my call but I would vote for "moving". Less risk of confusion.